ui/apps/platform/src/Containers/AccessControl/AccessControl.tsx

```diff
@@ -18,11 +19,50 @@ function AccessControl(): ReactElement {
     // TODO is read access required for all routes in improved Access Control?
     // TODO Is write access required anywhere in classic Access Control?
     const { hasReadAccess } = usePermissions();
-    const hasReadAccessForAuthProvider = hasReadAccess('Access');
+    const hasReadAccessForAccessControlPages = hasReadAccess('Access');
 
     return (
         <>
-            {hasReadAccessForAuthProvider ? (
+            <Alert
+                isInline
+                variant="warning"
+                title={
+                    <>
+                        <p>Next access resources will be replaced:</p>
+
+                        <p>
```

Comment:
Maybe a list for readability?

Reply:
When I try to use a list within this `Alert` component, it does not show up as a list. Wanted to ask @pedrottimark or @vjwilson if they know what might be the reason for this behaviour.

Reply:
> When I try to use a list within this `Alert` component, it does not show up as a list. Wanted to ask @pedrottimark or @vjwilson if they know what might be the reason for this behaviour.

There were were pretty strict style resets in pre-Red Hat style sheet, and we can't remove them until all the old style pages have been updated. We'd probably need to add a specific override in this file until then, to make a list show in this component.

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/css/trumps.css